default/stringtables/en.txt

```diff
@@ -1385,6 +1385,18 @@ Minimum distance monsters start from Capital
 RULE_MINIMUM_MONSTER_DISTANCE_CAPITAL_DESC
 Minimum distance monsters start from Capital
 
+RULE_OWNED_PLANETS_AT_MAXIMUM_UPKEEP
+Owned colonies at maximum upkeep
+
+RULE_OWNED_PLANETS_AT_MAXIMUM_UPKEEP_DESC
+Number of owned colonies at which upkeep per colony is maximum.
+
+RULE_END_GAME_COLONY_IP_OUTPUT
+IP output per colony at end game
```

Comment:
This is confusingly worded, I think...   Is it rather the upkeep cost per colony at end game?

Reply:
I'll change it by
```
RULE_MAXIMUM_COLONY_UPKEEP
Maximum upkeep per colony
```